api-report/cell.api.md

```diff
@@ -46,7 +46,8 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
     // (undocumented)
     protected loadCore(storage: IChannelStorageService): Promise<void>;
     protected onDisconnect(): void;
-    protected processCore(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown): void;
+    protected processCore(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: any): void;
+    protected rollback(op: any, localOpMetadata: any): void;
```

Comment:
The signatures should match the methods they are overriding from SharedObjectBase. I.e. localOpMetadata type should be unknown and parameter "op" should be "content". You'll just have to cast from unknown.

Reply:
Its solved but I have some concern about variable naming convention https://github.com/microsoft/FluidFramework/pull/12273/files#r1023015264

Reply:
Don't forget to update the rollback parameter name to "content"